views/tally.html

```diff
@@ -1,320 +1,44 @@
-			width: 50%;
-			max-width: 50%;
-			box-sizing: border-box;
-			margin: auto;
-			border: 1px solid #aaa;
-			box-shadow: 0 1px 0 1px rgba(0,0,0,.04);
-			border-radius: .5em;
-			-moz-appearance: none;
-			-webkit-appearance: none;
-			appearance: none;
-			background-color: #fff;
-			background-image: url('data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%22292.4%22%20height%3D%22292.4%22%3E%3Cpath%20fill%3D%22%23007CB2%22%20d%3D%22M287%2069.4a17.6%2017.6%200%200%200-13-5.4H18.4c-5%200-9.3%201.8-12.9%205.4A17.6%2017.6%200%200%200%200%2082.2c0%205%201.8%209.3%205.4%2012.9l128%20127.9c3.6%203.6%207.8%205.4%2012.8%205.4s9.2-1.8%2012.8-5.4L287%2095c3.5-3.5%205.4-7.8%205.4-12.8%200-5-1.9-9.2-5.5-12.8z%22%2F%3E%3C%2Fsvg%3E'),
-			linear-gradient(to bottom, #ffffff 0%,#e5e5e5 100%);
-			background-repeat: no-repeat, repeat;
-			background-position: right .7em top 50%, 0 0;
-			background-size: .65em auto, 100%;
-		}
-		.select-css::-ms-expand {
-			display: none;
-		}
-		.select-css:hover {
-			border-color: #888;
-		}
-		.select-css:focus {
-			border-color: #aaa;
-			box-shadow: 0 0 1px 3px rgba(59, 153, 252, .7);
-			box-shadow: 0 0 0 3px -moz-mac-focusring;
-			color: #222;
-			outline: none;
-		}
-		.select-css option {
-			font-weight:normal;
-		}
-		
-		.imgLogo {
-			margin: auto;
-			display: block;
-			max-width: 50%;
-			max-height: 50%;
-		}
-		
-		#divTally {
-			display: none;
-			width: 100%;
-			height: 100%;
-		}
-
-		.tallyDeviceTitle {
-			text-align: center;
-			color: #eeeeee;
-			font-size: 5vmin;
-			font-weight: bold;
-			width: 100%;
-			display: block;
-		}
+<!doctype html>
+<html lang="en">
 
-		.tallyDeviceDescription {
-			text-align: center;
-			color: #aaaaaa;
-			font-size: 4vmin;
-			width: 100%;
-			display: block;
-		}
-
-		.flash {
-			-webkit-animation:	screen-flash 500ms 5; /* Safari 4+ */
-			-moz-animation:		screen-flash 500ms 5; /* Fx 5+ */
-			-o-animation:		screen-flash 500ms 5; /* Opera 12+ */
-			animation:			screen-flash 500ms 5; /* IE 10+, Fx 29+ */
-		}
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="lib/css/bootstrap.min.css">
+  <title>Tally Arbiter</title>
+</head>
 
-		@-webkit-keyframes screen-flash {
-		0%, 49% {
-			background-color: rgb(255, 255, 255);
-		}
-		50%, 100% {
-			background-color: rgb(0, 0, 0);
-		}
-		}
-</style>
+<body>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <div class="container">
+      <a class="navbar-brand" href="#"><img src="lib/img/logo_white.png" height="30" alt="" loading="lazy"></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="navbar-toggler-icon"></span> </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
```

Comment:
Overall, I like the nav bar. I think it's really helpful. But I have my doubts about having it on the tally page as I don't think web tally viewers will need to access any other portions of the site. Thoughts?

Reply:
I agree, but when on desktop there should be a way to navigate further without hitting the back button. Maybe check viewport size and only display the white logo when on mobile?

Reply:
Yes, modify so that the nav bar is not displayed when on mobile. The logo can redirect to the index page when clicked/tapped.

Reply:
Should be pushed now